### PR TITLE
attempt to add sponsor logos to footer

### PR DESCRIPTION
**Pull request by jganseman**
Needs review and local testing!

- sponsor logos added in images dir for contestInterface and teacherInterface
- divFooter expanded (similar to divHeader).
- now hardcoded links. Once tested inline and style is OK, needs to be moved to translation.json like beoi logos for easy future replacement.

### CODE REVIEW

**Comment by smadbe** [comment]
shouldn't we be sure they sign first ?

**Comment by jganseman** [comment]
Sure, it's only meant to be merged when contract is finalized. But could maybe already be reviewed as I coded this blindly without setting up a PHP server


Op 21 jan. 2017 7:18 p.m. schreef Damien Leroy <notifications@github.com>:

shouldn't we be sure they sign first ?

File: teacherInterface/index.php [hunk 1]
```diff
@@ -262,6 +262,18 @@
...
+   <div class="header_sep_bottom"></div>   
+   <div style="z-index: 10; position:absolute; bottom: 0; right:0; font-variant: small-caps; margin-left: auto; margin-right: auto; text-align:center">
+      <table cellpadding="5px"><tr>
+      	<td style="padding: 0px 10px;">Supported by :</td>
+      	<td style="padding: 0px 10px;"><a href="http://www.docbyte.com"><img src="images/sponsor-1.png" style="height:45px" /></a></td>
+      	<td style="padding: 0px 10px;"><a href="http://www.exellys.com"><img src="images/sponsor-2.png" style="height:45px" /></a></td>
+      </table>	
+   </div>
+</div>   
+
```
Comment (smadbe) [inline teacherInterface/index.php:276]: not ok, floating on top and hiding instructions

File: contestInterface/index.php [hunk 2]
```diff
@@ -312,6 +312,14 @@
...
    <button type="button" id="buttonCloseNew" class="buttonClose" data-i18n="questions_finish_early" onclick='tryCloseContest()'></button>
+   
+   <div style="z-index: 10; position:absolute; bottom: 0; right:0; font-variant: small-caps; margin-left: auto; margin-right: auto; text-align:center">
+      <table cellpadding="5px"><tr>
+      	<td style="padding: 0px 10px;">Supported by :</td>
+      	<td style="padding: 0px 10px;"><a href="http://www.docbyte.com"><img src="images/sponsor-1.png" style="height:45px" /></a></td>
+      	<td style="padding: 0px 10px;"><a href="http://www.exellys.com"><img src="images/sponsor-2.png" style="height:45px" /></a></td>
+      </table>	
+   </div>
 </div>
```
Comment (smadbe) [inline contestInterface/index.php:323]: Not displayed. 

But anyway, should not be displayed in the actual contest interface, I'm afraid it interferes in some browsers/config.

**Review by smadbe — CHANGES_REQUESTED** [review]

**Comment by smadbe** [comment]
done in other branches

**Pull request closed without merging**